src/gameplay/voidGameTracker.ts

```diff
@@ -0,0 +1,36 @@
+import Game from './game';
+import { Alliance } from './types';
+
+export class VoidGameTracker {
+  private game: Game;
+  private playersVoted: Set<string> = new Set();
+
+  constructor(game: Game) {
+    this.game = game;
+  }
+
+  // Adds player to playersVoted set
+  // Returns true if numPlayersVoted > votesNeeded
```

Comment:
Can you remove line 12 and update line 13 to be something like:

> Returns true if the game should now be voided

Reply:
Done in https://github.com/vck3000/ProAvalon/pull/623/commits/9a4498cbabcfc22221a241ce0d5d9239ed86be38